_posts/2022-02-17-gsoc.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Google Summer of Code 2022"
+author: Kai Xu and the TuringLang team
+date: 2022-02-17
+---
+
+It's another year of the [Google Summer of Code](https://summerofcode.withgoogle.com/) time and we've compiled an updated list of exciting Turing proects!
+Below is a list of projects the Turing team would be interested in working with students on for the summer.
```

Comment:
Maybe:
Projects that the Turing team would be interested in working with students on over the summer are listed below.